config/server-api-list.json

```diff
@@ -0,0 +1,14 @@
+{
+  "versions": [
+    {
+      "version": "v1.0",
+      "api-list": {
+        "/get/browse/question-list": "/api/v1.0/TODO",
```

Comment:
Really uncertain about the best way to format the keys in the api-list. I think a filesystem format makes organization easier (using - instead of / would work too, it's just strings), but I'm not sure of the order of the directories. Some possibilities:

/get/browse/question-list
/browse/question-list/get
/browse/questions/list/get
etc.

Thoughts?

Reply:
I think /browse/question-list/get is better in my opinion. Not sure what other people think.

Reply:
I would agree. It makes more sense to divide root routers by functionality (browse/random/etc) than by type (get/post). For now, question-list is fine, but it may make more sense to have a questions router if more question endpoints are needed (individual question post, question responses)